package.json

```diff
@@ -11,18 +11,19 @@
   },
   "license": "MPL-2.0",
   "dependencies": {
+    "brave-core-ios": "https://github.com/brave/brave-browser/releases/download/v1.22.71/brave-core-ios-1.22.71.tgz",
     "page-metadata-parser": "^1.1.3",
     "readability": "github:mozilla/readability#b9f47bcc8d3c223cabe2dec6a42eeb3bd778d85c",
-    "webpack-cli": "^3.3.10",
-    "brave-core-ios": "https://github.com/brave/brave-browser/releases/download/v1.22.71/brave-core-ios-1.22.71.tgz"
+    "webpack-cli": "^3.3.10"
   },
   "devDependencies": {
     "babel-core": "^6.26.3",
     "babel-loader": "^7.1.5",
     "babel-preset-env": "^1.7.0",
     "babel-preset-es2015": "^6.24.1",
     "glob": "^7.1.6",
-    "webpack": "^4.42.0",
-    "mkdirp": "^1.0.3"
+    "mkdirp": "^1.0.3",
+    "url": "^0.11.0",
```

Comment:
This package had to be added, since webpack 5+ it is no longer bundled there I believe